### ISSUE TITLE
Make install fails on debian wheezy

### ISSUE THREAD

**Issue by mbrt** [vim/vim#772]
I have a `make install` failure on my machine (which I didn't a week ago):

```
./configure --with-features=huge --enable-luainterp --enable-gui=no --without-x --prefix=/usr
make VIMRUNTIMEDIR=/usr/share/vim/vim74
make install
```

results in:

```
/bin/sh: 6: -q: not found
```

The problem seems related to changes in 4adfaabfe7e07da9546b45130cad0d266ba48611. In particular this addition in `src/Makefile`:

```
if test -z "$(DESTDIR)" -a -x $(UPDATE_DESKTOP_DATABASE); then \
    $(UPDATE_DESKTOP_DATABASE) -q $(DESKTOPPATH); \
fi \
```

My problem seems coming from an empty `$(UPDATE_DESKTOP_DATABASE)` variable. Any clue?

**Comment by chrisbra** [comment]
Yes, this is probably caused by patch 7.4.1767 and a patch has already been provided [here](https://groups.google.com/d/msg/vim_dev/ruRV8HGj_Sg/wDPKYChhAQAJ) Please test this patch and report back on the mailinglist. Thanks.

**Comment by mbrt** [comment]
The patch resolves the issue indeed. I will report it to the mail list.

**Comment by chrisbra** [comment]
thanks. I think it has already been included as [7.4.1772](https://github.com/vim/vim/commit/54a38415caa31de8b4ac46a9234ae339b1fd6255)
